packages/thread-comment-ui/src/views/thread-comment-editor/index.tsx

```diff
@@ -148,3 +156,15 @@ export const ThreadCommentEditor = forwardRef<IThreadCommentEditorInstance, IThr
         </div>
     );
 });
+
+export const ThreadCommentSuggestion = ({ active, user }: { active: boolean; user: IUser }) => (
+    <div
+        className={clsx(
+            'univer-flex univer-items-center univer-text-sm univer-text-black',
+            { 'univer-bg-gray-50': active }
+        )}
+    >
+        <img className="univer-mr-[6px] univer-h-6 univer-w-6 univer-rounded-full" src={user.avatar} />
```

Comment:
How about setting the `img` tag to `draggable={false}` here to prevent users from accidentally dragging it?